src/Lifecycle/index.js

```diff
@@ -51,6 +51,7 @@ const stateChange = (state) => {
 
 // public API
 export default {
+  ready() { stateChange('background') },
```

Comment:
why does ready result into a background state change?

Reply:
Because it's up to the platform/transport layer to decide when it goes into active. might be a hot launch, might be a cold launch.

Reply:
but this is not the transport layer, this is the SDK. By doing the above you hardcode the ready() api to always set the state to background, effectively nullifying the usage of the ready() API.

Agree this should be done in the transport layer it, not in here. These are just the APIs and they should map accordingly, e.g. ready should set ready as a state.

Reply:
agreed, I let a specific feature leak into the SDK defaults. will fix.